cerberus/client.py

```diff
@@ -481,7 +481,7 @@ def put_file(self, secure_data_path, filehandle, content_type=None):
         """
 
         # Parse out the filename from the path
-        filename = secure_data_path.rsplit('/', 1)
+        filename = secure_data_path.rsplit('/', 1)[-1]
```

Comment:
I'm a bit surprised this fell through the cracks in my testing.

Though now that I look, we don't have any unit tests for this use-case.  I can see about adding those as a different PR.